go/rpc/client.go

```diff
@@ -49,3 +52,17 @@ type Client interface {
 	// Stop closes the client.
 	Stop()
 }
+
+// ConvertToCallMsg converts the interface to a *ethereum.CallMsg
+func ConvertToCallMsg(callMsgInterface interface{}) (*ethereum.CallMsg, error) {
```

Comment:
Is this only used in wallet extension? Not sure it's the right place for it.

Reply:
Yeah wasn't sure about that too, any suggestion on where it might be ?

Reply:
In with the wallet extension code I guess as a util. Then if wallet extension got thrown away one day, it would get thrown away too :)

Reply:
Yeah, good point - piggy backed it in as a helper function.